### PR TITLE
Df single bparser

### PR DESCRIPTION
**Pull request by dflanderka**
- remove matrix of parsers in FieldFormula
- fix yaml test files

### CODE REVIEW

File: tests/25_solute_dg_bc/01_inflow.yaml [hunk 1]
```diff
@@ -32,7 +32,7 @@ problem: !Coupling_Sequential
           bc_conc: 3
         - region: domain
           diff_m: !FieldFormula
-            value: ' ((0) if (y>0.5) else (1e-2)) '
+            value: '[ [((0) if (y>0.5) else (1e-2)), 0, 0], [0, ((0) if (y>0.5) else (1e-2)), 0], [0, 0, ((0) if (y>0.5) else (1e-2))] ]'
```
Comment (jbrezmorf) [inline tests/25_solute_dg_bc/01_inflow.yaml:35]: Rewrite as "scalar * eye(3)", use manual unit tensor until `eye` is supported by bparser.

File: unit_tests/fields/field_algo_base_test.cpp [hunk 2]
```diff
@@ -534,11 +534,11 @@ string field_input = R"INPUT(
    init_conc=[ 10, 20, 30],    // FieldConst
    conductivity={ //3x3 tensor
```
Comment (jbrezmorf) [inline unit_tests/fields/field_algo_base_test.cpp:535]: Revisit. Should be tensor here. Does the test pass?

**Comment by dflanderka** [comment]
Fix field_python_test after merge of master

File: unit_tests/fields/field_algo_base_test.cpp (hunk 2 above)
Comment (dflanderka) [reply unit_tests/fields/field_algo_base_test.cpp:535]: Input string is used in init_from_input test without assemblation. Test passes with both (vector and tensor) formula. But I changed input formula to tensor according to the comment.

**Pull request merged**